### PR TITLE
sendPaymentLocal and protocol-only buildPaymentLocal

### PR DESCRIPTION
**Pull request by mlsteele**

### CODE REVIEW

File: protocol/avdl/stellar1/local.avdl [hunk 1]
```diff
@@ -131,6 +131,110 @@ protocol local {
...
+    string subtext; // Examples: "", "max doesn't accept HUGZ"
+  }
+
+  // Check but do not send a payment.
+  // Opt  - Optional field.
+  // Opt* - Optional for this RPC but required for res.readyToSend to be set.
+  BuildPaymentResLocal buildPaymentLocal(
+    AccountID from,
+    // Opt. Stellar seqno of the sender when the user started the form.
+    // If it is old, the payment will be blocked blocked.
```
Comment (chrisnojima) [inline protocol/avdl/stellar1/local.avdl]: blocked blocked?

File: protocol/avdl/stellar1/local.avdl [hunk 2]
```diff
@@ -131,6 +131,110 @@ protocol local {
...
+    string fromSeqno,
+    // Opt*. Recipient username, acctID, or assertion.
+    // Examples: "", "russel", "GBQTE2V7Y356TFBZL6YZ2PA3KIILNSAAQRV5C7MVWS22KQTS4EMK7I46"
+    string to,                                    
+    bool toIsAccountID,
```
Comment (chrisnojima) [inline protocol/avdl/stellar1/local.avdl]: should this be some kind of enum instead

File: protocol/avdl/stellar1/local.avdl [hunk 3]
```diff
@@ -131,6 +131,110 @@ protocol local {
...
+    string amount, // Opt*. Amount of the asset OR outside currency being sent. Example: "3.005"
+    // Zero or one of `asset` and `currency` should be set.
+    // If both are null or `asset` is native XLM then XLM are being sent.
+    // If `currency` is set then XLM is being sent after converting `amount`.
+    union { null, OutsideCurrencyCode } currency,
+    union { null, Assset } asset,
```
Comment (chrisnojima) [inline protocol/avdl/stellar1/local.avdl]: Assset

File: protocol/avdl/stellar1/local.avdl (hunk 2 above)
Comment (mlsteele) [reply protocol/avdl/stellar1/local.avdl]: Username and assertion go mostly through the same code paths. Probably only these two will be used anyhow. Should be easy to change

**Comment by mlsteele** [comment]
@chrisnojima did those things. Also @cjb took a look. Repurposing this PR for the implementation of send.

**Comment by patrickxb** [comment]
Sorry, didn't know this was ready for review...Will look.

**Pull request merged**